src/components/home/home.scss

```diff
@@ -2,58 +2,136 @@
     HERO BLOCK
 *******************************/
 $bg-img: url('../../images/meal-prep.jpg');
-
 $teal-empahasis-color: #01B4BC;
 $red-cta-color: #FA5457;
 
 
-
 #hero-block {
-  background-image: $bg-img;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: $bg-img no-repeat center center;
   background-size: cover;
+  width: 100%;
+  height: 100%;
 
   .cta-block {
+    // width: 400px;
+    box-sizing: border-box;
     background-color: #ffffffd8;
-    padding: 200px 50px;
     text-align: center;
-    
-    @media only screen and (min-width: 700px) {
-      padding: 20vw 10vw;
-    }
+    height: 100%;
+    width: 100%;
+    padding: 10%;
+      h1 {
+        padding-top:50px;
+        font-size: 1.8rem;
+        font-weight:300;
+      }
 
-    h1 {
-      font-size: 70px;
+      p {
+        font-size: 1.1rem;
+        font-family: Verdana, Geneva, Tahoma, sans-serif;
+      }
 
-      @media only screen and (min-width: 700px) {
-        font-size: 30px;
+      .cta-btn {
+        background-color: #FA5457;
```

Comment:
Let's add use the variables for colors at the top so they are easy to update altogether.